### PR TITLE
adding Florentine Swords to the list

### PR DESCRIPTION
**Pull request by VMSolidus**
Florentine swords are a specialized fighting style from HEMA that uses a sword and parrying blade. It's a primarily defensive style that is difficult to perform, but has an advantage of allowing a swordsman to compete with blades that outclass him(Such as going up against a Zweihander user). I would have added the parrying-dagger seperately, but since Dual-wielding isn't a thing in this game, I'll have to settle for it being a single item that uses both. 

Inferior variant has also been added, for the purpose of diluting drop rates.

https://en.wikipedia.org/wiki/Parrying_dagger

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
We really need the option to wield two weapons at once; hacky stuff like this isn't really the best way to go.

**Comment by evilexecutive** [comment]
Kevin doesn't want dual-wielding, because it's impractical. HEMA shows that it is actually pretty practical, but only in one very particular case(Sword and parrying dagger)... So yea..

**Comment by Coolthulhu** [comment]
Hacks like that belong in a mod, not mainline.

HEMA is mod content anyway.

**Comment by Rivet-the-Zombie** [comment]
>HEMA is mod content anyway.

I'd be in favor of mainlining it **if we ever get a good enough version that it could compete with the other melee styles**; there's no reason to favor Eastern weapon styles exclusively.

**Comment by Mecares** [comment]
To be honest it is not the crazy dual Katana stuff we need dual wielding for it is stuff like: Sword and Shield, Gun and Sword, Torch and Whip, Gun and Riot Shield...

**Comment by evilexecutive** [comment]
Yea, Two arming swords or katanas is amazingly impractical, but CAN work with the right training. Despite that, in most cases with two swords, the user will never truly be able to swing both at the same time unless they're ambidextrous(We can add that as a trait if two-hands is a thing!). The most it would be useful for is two weapons that serve completely different purposes, such as Sword and parrying blade, sword and gun, and the other stuff mecares stated.

**Comment by evilexecutive** [comment]
Also... How do I add another file to this pull request? I still don't know how to do that...

**Comment by evilexecutive** [comment]
Nevermind, that was oddly complicated, but I somehow figured it out!

**Comment by Coolthulhu** [comment]
Once again: this should be in a mod. Medieval one would fit well.

If it is to be in the mainline, it will need to wait until we support dual wielding without horrible hacks.

File: data/json/items/melee.json [hunk 1]
```diff
@@ -1916,6 +1916,42 @@
     "category" : "weapons",
     "qualities": [["CUT", 1], ["BUTCHER", 8]]
   },{
+    "id": "florentine_swords",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "florentine swords",
+    "description": "This is a mated pair of arming sword and parrying dagger, primarily used for florentine fighting. The dagger is made of a sturdy material, and has implements intended to disarm and block",
```
Comment (Night-Pryanik) [inline data/json/items/melee.json]: Two spaces after periods, please. Same below.

**Comment by mugling** [comment]
Closed in agreement with @Coolthulhu's comments. Would be welcomed as mod content or otherwise please resubmit when (and if) we support dual wielding.

**Comment by kevingranade** [comment]
I don't want stereotypical dual-wielding that implies "two swords is twice
as effective as one".  See
http://smf.cataclysmdda.com/index.php?topic=10444.msg247535#msg247535 for
more detail.

**Pull request closed without merging**